### PR TITLE
Update setup.rst (version 6.1)

### PR DESCRIPTION
**Pull request by BahmanMehrdad**

### CODE REVIEW

File: setup.rst [hunk 1]
```diff
@@ -273,14 +273,14 @@ stable version. If you want to use an LTS version, add the ``--version`` option:
     $ symfony new my_project_directory --version=next
 
     # you can also select an exact specific Symfony version
-    $ symfony new my_project_directory --version=6.0
+    $ symfony new my_project_directory --version=6.0.*
```
Comment (OskarStark) [inline setup.rst:276]:
```suggestion
    $ symfony new my_project_directory --version=6.1.*
```

File: setup.rst [hunk 2]
```diff
@@ -273,14 +273,14 @@ stable version. If you want to use an LTS version, add the ``--version`` option:
...
 
 The ``lts`` and ``next`` shortcuts are only available when using Symfony to
 create new projects. If you use Composer, you need to tell the exact version:
 
 .. code-block:: terminal
 
-    $ composer create-project symfony/skeleton:"^6.0" my_project_directory
+    $ composer create-project symfony/skeleton:"6.0.*" my_project_directory
```
Comment (OskarStark) [inline setup.rst:283]:
```suggestion
    $ composer create-project symfony/skeleton:"6.1.*" my_project_directory
```

Comment (OskarStark) [inline setup.rst]: These two changes should be reverted, its about the LTS version, which is still 5.4

**Review by OskarStark — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you @BahmanMD.

**Pull request merged**